app/models/float_field_type.rb

```diff
@@ -0,0 +1,39 @@
+class FloatFieldType < FieldType
+  attr_accessor :float
+ 
+  validates_numericality_of :float
```

Comment:
So this could cause the validations to occur when they're not supposed to, like we talked about. Could you please add a check here for the key `:presence` to see if this should be run.

Additionally, could you add a check to run the presence validator (`validate presence: true, if: Proc.new { |x| validate_key(:presence) }`), otherwise the validations' errors would be unhelpful to a User (they would be told their input is not numerical when it needs to be present)